src/services/JiraService.js

```diff
@@ -43,11 +44,34 @@ module.exports.assigneeInfo = (jql) => {
     })
     .catch(error => {
         console.log("Jira Error: " + error.response.data['errorMessages']);
-        throw new Error(error.response.data['errorMessages'] || error.response.data['warningMessages']);
+        throw new Error(error.response.data['errorMessages']);
+    });
+};
+
+// search the JIRA API using a project key and return the response or throw and error
+module.exports.projectInfo = (projectKey) => {
+    console.log('JIRA URL: ' + jiraDetails.url + jiraDetails.project_endpoint + projectKey);
+    return axios.get(jiraDetails.url + jiraDetails.project_endpoint + projectKey, {
+        auth: {
+            username: jiraDetails.name,
+            password: jiraDetails.password
+        }
+    })
+    .then(response => {
+        return response.data;
+    })
+    .catch(error => {
+        console.log("Jira Error: " + error.response.data['errorMessages']);
+        throw new Error(error.response.data['errorMessages']);
     });
 };
 
 // Given a JIRA issueID as input, return it as a Slack hyperlink
 module.exports.HyperlinkJiraIssueID = (issueID) => {
-    return `<${process.env.JIRA_URL}/browse/${issueID}|${issueID.toUpperCase()}>`
+    return `<${process.env.JIRA_URL}/browse/${issueID}|${issueID}>`
+};
```

Comment:
Since you have created the object with constants in it, do you want to use `JiraDetails` here rather than `process.env.JIRA_URL` ?

Reply:
Oh good pick-up, complete oversight on my behalf!